ntbs-service/DataMigration/NotificationMapper.cs

```diff
@@ -904,5 +900,12 @@ private void SetDenotificationReasonAndDescription(MigrationDbNotification notif
                                                          (notification.DenotificationDate?.ToString() ?? "missing");
             }
         }
+
+        private static int? GetCountryTotalOrNull(Status? status, int? countryCount, int countryCount2)
```

Comment:
Maybe we could be a little more descriptive with these names... perhaps `totalEnteredByUser` and `countryCount`?

Reply:
Have gone with those names 👍